packages/mobile/src/geth/networkConfig.ts

```diff
@@ -89,6 +89,7 @@ const networkConfigs: { [testnet: string]: NetworkConfig } = {
     nodeDir: `.${Testnets.alfajores}`,
     syncMode: DEFAULT_SYNC_MODE,
     initiallyForno: FORNO_ENABLED_INITIALLY,
+    // blockchainApiUrl: 'http://127.0.0.1:8080',
```

Comment:
just checking that you want to keep this?

Reply:
Yeah, it's just commented out. It saves a little time when testing locally so that I only need to uncomment this and comment the other line instead of writing the whole URL from scratch. Not a huge deal of course, if you don't like it I can delete it, it's just a minor convenience thing.

Reply:
i don't really have strong feelings about this (my eye only twitched once!), just wanted to check :) if it is saving you time then i don't mind it being here